frontend/src/components/DrawerCollectionList.tsx

```diff
@@ -41,6 +42,18 @@ const DrawerCollectionListItem: React.FC<DrawerCollectionListItemProps> = observ
     return store.drawerSelectedCollection === collectionId
   }
 
+  function hasWhiteSpace(text: string): boolean {
+    return /\s/g.test(text);
+  }
+
+  function wrapTextWithoutSpaces(text: string): string {
+    if (hasWhiteSpace(text)) {
+      return text
+      } else {
+        return `${text.substring(0, 30)}...`
```

Comment:
it adds "..." to every keyword name, that does not have whitespace including short names like "Setup"

Reply:
@MarcinMaciaszek, @nojas10 Could You check behaviour of those changes on real life corpo repo? ;)